.gitignore

```
@@ -0,0 +1 @@
+vendor
```

```suggestion
/vendor/
```
shouldnt it be the folder reference?

---

yes bhai